data-es6.js

```diff
@@ -17273,6 +17273,45 @@ exports.tests = [
         jerryscript2_0: false,
       }
     },
+    {
+      name: 'in property key definitions',
+      exec: function(){/*
+        var o = { \u{102C0} : 2 };
+        return o['\ud800\udec0'] === 2;
+      */},
+      res: {
+        ejs: true,
```

Comment:
I reused the same test results, since this was split from the previous test. I imagine everything will pass them both individually.